database/schema.sql

```diff
@@ -140,3 +140,6 @@ GRANT SELECT ON analysis, certificates, scans, trust TO tlsobsscanner;
 GRANT INSERT ON analysis, certificates, scans, trust TO tlsobsscanner;
 GRANT UPDATE ON analysis, certificates, scans, trust TO tlsobsscanner;
 GRANT USAGE ON analysis_id_seq, certificates_id_seq, scans_id_seq, trust_id_seq TO tlsobsscanner;
+
+-- # Migrations
+ALTER TABLE certificates ADD COLUMN x509_extendedKeyUsageOID jsonb NULL;
```

Comment:
I think we should put this is a separate file under `database/migrations/` and label it with the release version (in this case, `1.3.3`)

Reply:
Are you thinking of putting manual migrations there until #318 is fixed?